lib/pharos/up_command.rb

```diff
@@ -110,8 +110,8 @@ def configure(config)
 
       craft_time = Time.now - start_time
       puts pastel.green("==> Cluster has been crafted! (took #{humanize_duration(craft_time.to_i)})")
-      puts "    You can connect to the cluster with kubectl using:"
-      puts "    export KUBECONFIG=~/.pharos/#{manager.sorted_master_hosts.first.api_address}"
+      puts "    To configure kubectl for connecting to the cluster, use:"
+      puts "      #{$PROGRAM_NAME} kubeconfig > ~/.kube/config"
```

Comment:
I think we still should show the `export KUBECONFIG` hint.

Reply:
`~/.kube/config` is not a good idea as an example because it will overwrite local kubeconfig that most likely is already there. Or did I miss something?

Reply:
yeah, I agree that it's a bad example